pkg/scheduler/factory/factory.go

```diff
@@ -1320,7 +1320,7 @@ func (c *configFactory) MakeDefaultErrorFunc(backoff *util.PodBackoff, podQueue
 					}
 				}
 			} else {
-				glog.Errorf("Error scheduling %v %v: %v; retrying", pod.Namespace, pod.Name, err)
+				glog.V(3).Infof("Error scheduling %v %v: %v; retrying", pod.Namespace, pod.Name, err)
```

Comment:
I'm actually not convinced that this one should be demoted.

Reply:
is there a reason you want to log it twice?

Reply:
The "info" log should be removed from the [`schedule` function](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/scheduler.go#L194) and this should remain as an "error".

Reply:
Sounds good. Will update the PR soon.